.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
   - 7.0
   - 7.1
   - 7.2
```

Comment:
Since tests are successful for PHP 7.2, you could remove it from allowed failures.

Reply:
done